recipe/meta.yaml

```diff
@@ -37,8 +44,13 @@ requirements:
     - cmake
     - eigen
     - gcc  # [linux]
+    - mpich
+    - petsc {{petsc_version}}
+    - petsc4py {{petsc4py_version}}
     - pkg-config
+    - ptscotch
     - zlib 1.2.*
+    - zlib
```

Comment:
This line should be deleted because of the pinned version that is already there.